### PR TITLE
Remove all spaces from potential appUserModelID as it isn't allowed by the spec

### PR DESCRIPTION
**Pull request by MarshallOfSound**
Pairs up with Squirrel/Squirrel.Windows/pull/778

### CODE REVIEW

File: lib/browser/init.js [hunk 1]
```diff
@@ -93,8 +93,8 @@ if (process.platform === 'win32') {
 
   if (fs.statSyncNoException(updateDotExe)) {
     var packageDir = path.dirname(path.resolve(updateDotExe))
-    var packageName = path.basename(packageDir)
-    var exeName = path.basename(process.execPath).replace(/\.exe$/i, '')
+    var packageName = path.basename(packageDir).replace(/ /g, '')
```
Comment (zeke) [inline lib/browser/init.js]: maybe `/\s /g` would match more space-like characters?

**Comment by zcbenz** [comment]
👍

**Comment by eugirdor** [comment]
Is this going to cause any issues with auto-updating the app?  I know there was a similar change made on macOS for the bundle identifier, and it caused auto-updating to fail because the new app's identifier no longer matched the old one.

**Comment by MarshallOfSound** [comment]
@eugirdor This shouldn't affect auto-updating the app as the appUserModelID is independent of how Squirrel triggers updates.

**Comment by eugirdor** [comment]
@MarshallOfSound Cool, thanks.

**Pull request merged**